packages/cloudflare/src/cli/commands/populate-cache.ts

```diff
@@ -92,13 +92,14 @@ export function getCacheAssets(opts: BuildOptions): CacheAsset[] {
   return assets;
 }
 
-function populateR2IncrementalCache(
+async function populateR2IncrementalCache(
   options: BuildOptions,
   populateCacheOptions: { target: WranglerTarget; environment?: string }
 ) {
   logger.info("\nPopulating R2 incremental cache...");
 
   const config = unstable_readConfig({ env: populateCacheOptions.environment });
+  const proxy = await getPlatformProxy<Record<string, string | undefined>>(populateCacheOptions);
```

Comment:
Would that work (also on line 118)
```suggestion
  const proxy = await getPlatformProxy<CloudflareEnv>(populateCacheOptions);
```